pandas/io/pytables.py

```diff
@@ -887,7 +887,9 @@ def remove(self, key, where=None, start=None, stop=None):
         where = _ensure_term(where, scope_level=1)
         try:
             s = self.get_storer(key)
-        except:
+        except KeyError as ke:
```

Comment:
Can this just be

```python
except KeyError:
    raise
```

Maybe w/ a comment explaining what's going on. Or is that python 3 only?

Reply:
@TomAugspurger : The syntax is supported in Python 2 and 3.  This would be a good simplification.

Reply:
I've just checked the Python 2 docs, and it seems to be compatible with that version as well (https://docs.python.org/2.7/tutorial/errors.html). If you prefer this style, I'll change the code and add the comment.

Reply:
This is pretty standard syntax IMO (we use in other places in other codebase), so I'm not sure how necessary commenting is.  That being said, couldn't hurt, so let's just comment it :)